theme/wds-headless-theme/acf-json/group_5ff605fe69e7d.json

```diff
@@ -0,0 +1,66 @@
+{
```

Comment:
Do we want to keep the ACF JSON bundled into the theme or break it out?

Reply:
I would rather it not be here.